fluss-common/src/main/java/org/apache/fluss/config/ConfigOptions.java

```diff
@@ -394,6 +394,14 @@ public class ConfigOptions {
                             "The rack for the tabletServer. This will be used in rack aware bucket assignment "
                                     + "for fault tolerance. Examples: `RACK1`, `cn-hangzhou-server10`");
 
+    public static final ConfigOption<Integer> TABLET_SERVER_IO_POOL_SIZE =
+            key("tablet-server.io-pool.size")
```

Comment:
Can we unify the io-pool for coordinator and tablet servrer? We can rename this config to `server.io-pool.size` in this PR. And create follow-up issues:
1. deprecate `kv.snapshot.transfer-thread-num` and `remote.log.data-transfer-thread-num`, and use `server.io-pool.size` instead, may turn it into 10 threads by default. 
2. deprecate `coordinator.io-pool.size` and use `server.io-pool.size` instead.